XmlFormatterOsIndependent/Update/LocalVersionRecieverStrategy.cs

```diff
@@ -0,0 +1,38 @@
+﻿using PluginFramework.EventMessages;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using XmlFormatterModel.Update;
+using XmlFormatterModel.Update.Strategies;
+
+namespace XmlFormatterOsIndependent.Update
+{
+    /// <summary>
+    /// Strategy to use to get the local version
+    /// </summary>
+    internal class LocalVersionRecieverStrategy : IVersionRecieverStrategy
+    {
+        /// <summary>
+        /// Was there any error in recieving the version
+        /// </summary>
+        public event EventHandler<BaseEventArgs> Error;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused event 'Error' or invoke it.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)